### DISCUSSION TITLE
Plex is not recognizing torrents folder on virtual drive

### DISCUSSION THREAD

**Discussion by smodekurti** [itsToggle/plex_debrid#38] [General]
First of all thank you very much for an amazing work... I tried setting this up on both my Mac and Raspberry Pi. It worked flawless on my Mac. However, i am not able to make it work on Raspberry Pi. The script works just fine, is able to monitor watchlists and download the cached torrents. My challenge is that when I setup Movies in Plex and map to virtual drive, Plex is not able to load the movie files from virtual folder. I could only map with the root folder and it doesn't scan underlying files under torrent folder. The same setup works just fine on Mac... I would appreciate help
Configuration on Mac
<img width="586" alt="Screen Shot 2022-07-14 at 5 38 09 PM" src="https://user-images.githubusercontent.com/1075463/179100050-a9f1e82d-0127-47b0-9455-cc12b113b59b.png">

Configuration on Rasp
<img width="590" alt="Screen Shot 2022-07-14 at 5 37 24 PM" src="https://user-images.githubusercontent.com/1075463/179100059-3f1dffd4-df76-4a16-aa25-2fe51a019838.png">

**Comment by itsToggle** [comment]
On a Raspberry Pi! Thats awesome :)

It could be that Linux requires some more permission-granting. You could check the official rclone documentation, but from the top of my head: 
- Have you tried adding "--allow-other" to your mounting command? This mounting tag should make the virtual folder visible to Plex.
- Also make sure that the linux user Plex is using has system access to the virtual folder

What command are you currently using to mount rclone? :)

**Reply by smodekurti** [reply]
Thanks for the suggestion.. i was able to make it work after adding "--allow-other" param to rclone command...

Thanks again

**Comment by coachacg** [comment]
> First of all thank you very much for an amazing work... I tried setting this up on both my Mac and Raspberry Pi. It worked flawless on my Mac. However, i am not able to make it work on Raspberry Pi. The script works just fine, is able to monitor watchlists and download the cached torrents. My challenge is that when I setup Movies in Plex and map to virtual drive, Plex is not able to load the movie files from virtual folder. I could only map with the root folder and it doesn't scan underlying files under torrent folder. The same setup works just fine on Mac... I would appreciate help Configuration on Mac <img alt="Screen Shot 2022-07-14 at 5 38 09 PM" width="586" src="https://user-images.githubusercontent.com/1075463/179100050-a9f1e82d-0127-47b0-9455-cc12b113b59b.png">
> 
> Configuration on Rasp <img alt="Screen Shot 2022-07-14 at 5 37 24 PM" width="590" src="https://user-images.githubusercontent.com/1075463/179100059-3f1dffd4-df76-4a16-aa25-2fe51a019838.png">

Did you do anything special to get your Mac to work? I can seem to get my Plex_Debrid to scrape. It acknowledges changes to my Watchlist but I continue getting the empty library error and nothing downloads.
